yarn.lock

```diff
@@ -1386,6 +1386,18 @@
   dependencies:
     tslib "~2.3.0"
 
+"@hapi/hoek@^9.0.0":
+  version "9.3.0"
+  resolved "https://registry.yarnpkg.com/@hapi/hoek/-/hoek-9.3.0.tgz#8368869dcb735be2e7f5cb7647de78e167a251fb"
+  integrity sha512-/c6rf4UJlmHlC9b5BaNvzAcFv7HZ2QHaV0D4/HNlBdvFnvQq8RI4kYdhyPCl7Xj+oWvTWQ8ujhqS53LIgAe6KQ==
+
+"@hapi/topo@^5.0.0":
```

Comment:
didn't find any reason to change yarn lock

Reply:
It changed after running yarn install. I don't know the actual reason of it.

Reply:
for now you can just copy the yarn.lock from base branch and keep that in your feature branch. this change will disappear.
if there's issue like you faced then it should be solved properly.